builtin/providers/aws/resource_aws_cloudtrail.go

```diff
@@ -165,3 +190,42 @@ func resourceAwsCloudTrailDelete(d *schema.ResourceData, meta interface{}) error
 
 	return err
 }
+
+func cloudTrailGetLoggingStatus(conn *cloudtrail.CloudTrail, id string) (bool, error) {
```

Comment:
If you'd turn the 2nd argument to `*string`, you can avoid the dereferencing above + creating the pointer via `aws.String()` below, but I it's not a blocker for the PR, rather a nitpick. :wink: